src/System.Private.ServiceModel/tests/Scenarios/Binding/Http/BasicHttpBindingTests.cs

```diff
@@ -4,17 +4,20 @@
 using System;
 using System.ServiceModel;
 using System.ServiceModel.Channels;
+using System.ServiceModel.Tests.Common;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using TestTypes;
 using Xunit;
 
-public static class Binding_Http_BasicHttpBindingTests
+public class Binding_Http_BasicHttpBindingTests : OuterLoopTests
```

Comment:
I'd like to push back a bit for 2 reasons:
1. One goal was for Bridge to not impact the tests themselves.  I thought our only point of contact with Bridge would be the existing (static) test helpers.  Taking over the base class for a set of tests is fairly impactful and something that needs some careful consideration. We may have other reasons to want a shared base class other than for the Bridge, so I'd hate to close off that option unless strictly necessary.
2.  I don't believe TestFixture is available to us in NET Native xunit. That version of xunit is severely pared back, so I predict that approach will not work when we run these tests in our NET Native testing environment.

I would like to hear what the benefits are of a base class or test fixture over a simple static singleton (BridgeClient?).